source/h2_stream.c

```diff
@@ -747,6 +747,10 @@ int aws_h2_stream_on_activated(struct aws_h2_stream *stream, enum aws_h2_stream_
     stream->thread_data.window_size_self =
         connection->thread_data.settings_self[AWS_HTTP2_SETTINGS_INITIAL_WINDOW_SIZE];
 
+    if (!connection->base.stream_manual_window_management) {
+        stream->thread_data.window_size_self_dropped_threshold =
+            connection->thread_data.settings_self[AWS_HTTP2_SETTINGS_INITIAL_WINDOW_SIZE] / 2;
+    }
     if (with_data) {
```

Comment:
trivial: whitespace after logical if-block please (or if/else block, or if/elseif/elseif block)
```suggestion
    }
    
    if (with_data) {
```